### DISCUSSION TITLE
Remove the IBM logo in the footer

### DISCUSSION THREAD

**Discussion by sid-r-singh** [carbon-design-system/carbon#8831] [Help] [answered]
I am using gatsby-theme-carbon. How do I remove the IBM logo present in footer ? There is no code corresponding to this image in [Footer.js](https://github.com/carbon-design-system/gatsby-theme-carbon/blob/main/packages/example/src/gatsby-theme-carbon/components/Footer.js).

**Comment by joshblack** [comment, marked as answer]
Hi there @sid-r-singh! 👋 

Since this is a Gatsby Theme question, it might be best to ask over on: https://github.com/carbon-design-system/gatsby-theme-carbon

This repo/project is used for tracking the Design System itself which is why this question may not have received a response quickly (sorry about that!)

Hope this helps!
